mongo/integration/collection_test.go

```diff
@@ -1737,11 +1737,11 @@ func TestCollection(t *testing.T) {
 			deletes := len(mt.GetAllStartedEvents())
 			assert.True(mt, deletes > 1, "expected multiple batches, got %v", deletes)
 		})
-		mt.Run("update with batches", func(mt *mtest.T) {
+		mt.RunOpts("update with batches", mtest.NewOptions().ClientType(mtest.Mock), func(mt *mtest.T) {
```

Comment:
Switched this test to use the mock client, as performing the bulkwrite was causing the test to time out

Reply:
Minor: This change looks good. I have a slight preference be in favor of making this change as part of GODRIVER-2070 so it is clear that this is not a requirement for adding 5.0 tests.